src/custom-element.js

```diff
@@ -550,13 +550,13 @@ function createBaseCustomElementClass(win) {
           layoutBox.top >= 0) {
           // Few top elements will also be pre-initialized with a loading
           // element.
-          getVsync(this).mutate(() => {
-            // Repeat "loading enabled" check because it could have changed while
-            // waiting for vsync.
-            if (this.isLoadingEnabled_()) {
+          if (this.resources_) {
+            this.getResources().mutateElement(this, () => {
               this.prepareLoading_();
-            }
-          });
+            });
+          } else {
+            this.prepareLoading_();
+          }
```

Comment:
A helper function would be clearer. You can also use it on line 1547 below.

```js
this.mutateIfConnected_(() => this.prepareLoading_());

// ...

mutateIfConnected(mutator) {
  if (this.resources_) {
    this.getResources().mutateElement(this, mutator);
  } else {
    mutator();
  }
}
```

Reply:
Done.